lib/jekyll/commands/doctor.rb

```diff
@@ -59,8 +60,24 @@ def conflicting_urls(site)
           conflicting_urls
         end
 
-        private
+        def fsnotify_buggy?(site)
+          return true if !Utils::Platforms.osx?
+          if Dir.pwd != `pwd`.strip
+            Jekyll.logger.error "  " + <<-STR.strip.gsub(/\n\s+/, "\n  ")
+              We have detected that there might be trouble using using
```

Comment:
Fixed, and because of that you forced me to read the rest of it as a double check and I noticed I did `docker` instead of `doctor` in my platforms addition.